templates/voting_platform/_layout_base.html.twig

```diff
@@ -10,7 +10,7 @@
 {% block designation_banner '' %}
 
 {% block election_sub_title %}
-    <h2>Bienvenue sur la plateforme de désignation par vote électronique de La République en Marche !</h2>
+    <h2>Bienvenue sur la plateforme de désignation par vote électronique de {% if app.request.httpHost == renaissance_host %}Renaissance{% else %}La République En Marche{% endif %} !</h2>
```

Comment:
could you check `is_renaissance_host` instead?

Reply:
yes